### DISCUSSION TITLE
HERO11 Mini Labs 2.20.70

### DISCUSSION THREAD

**Discussion by dnewman-gpsw** [gopro/labs#288] [Announcements]
Now both HERO11 cameras have the new enhance Labs command set. So this is a biggish update for Mini (it did have some of this in 2.10.70).

Example of the new scripting commands: 
- combination triggers https://www.youtube.com/shorts/dNrCaV932FI
- remote proximity trigger https://youtube.com/shorts/iemaVm2cQek

Mini did get two extra commands (these will eventually port to HERO11, other see note below)

The **!Cx** where x is the GBs to make sure are clear on your Media. If more than x is available, no files are removed. x is limited to half SD size, so your recent captures are always preserved. Deletes the oldest media first. Ideal for dashcam applications. 
The **!X** to exit a running script. >l99!X!S!2R <- loop for 99 times, then exit.  This was an omission in the command language, simplifies exiting a script.

The Release notes are updated here: https://gopro.github.io/labs/control/notes/
New firmware : https://gopro.github.io/labs/

Note: Will any of this port to HERO10 and MAX, both are likely, maybe toward the end of the year. So not soon. As for other cameras HERO9 and HERO8 etc. unlikely, as these cameras are no longer sold they don't typically get many updates.

**Comment by HiStephenT** [comment]
Hi, does this add back the GCMP or should I still use ver71

**Reply by dnewman-gpsw** [reply]
GCMP is supported in 2.20.70.

**Reply by HiStephenT** [reply]
Perfect thanks, I didn’t see it in the changelogs.
